src/main/java/uk/gov/hmcts/darts/transcriptions/api/TranscriptionsApi.java

```diff
@@ -10,6 +10,4 @@ public interface TranscriptionsApi {
 
     UpdateTranscriptionResponse updateTranscription(Integer transcriptionId, UpdateTranscription updateTranscription);
 
-    void closeTranscriptions();
```

Comment:
Author note:
this method has been moved to a newly introduced "processor" component. This is similar to how the other automated tasks are structured. They delegate to a processor.